daemon/tests/happy_path.rs

```diff
@@ -152,6 +156,40 @@ async fn collaboratively_close_an_open_cfd() {
     assert_next_state!(CfdState::Closed, maker, taker, order_id);
 }
 
+#[tokio::test]
+async fn force_close_an_open_cfd() {
+    let _guard = init_tracing();
+
+    let oracle_data = OliviaData::example_0();
+    let (mut maker, mut taker, order_id) =
+        start_from_open_cfd_state(oracle_data.announcement()).await;
+
+    // Taker initiates force-closing
+    taker.force_close(order_id).await;
+
+    deliver_event!(maker, taker, Event::CommitFinality(order_id));
+    sleep(Duration::from_secs(5)).await; // need to wait a bit until both transition
+
+    // Delivering the wrong attestation does not move state to `PendingCet`
+    deliver_event!(maker, taker, dummy_wrong_attestation());
+
+    assert_next_state!(CfdState::OpenCommitted, maker, taker, order_id);
+
+    deliver_event!(maker, taker, oracle_data.attestation());
+
+    deliver_event!(maker, taker, Event::CetTimelockExpired(order_id));
+
+    sleep(Duration::from_secs(5)).await;
+
+    assert_next_state!(CfdState::PendingCet, maker, taker, order_id);
+
+    deliver_event!(maker, taker, Event::CetFinality(order_id));
+
+    sleep(Duration::from_secs(5)).await; // need to wait a bit until both transition
+
+    assert_next_state!(CfdState::Closed, maker, taker, order_id);
```

Comment:
A nit on test structure, I find excessive use of newlines actually decreases readability. What do you think of this:

```suggestion
    // Delivering the wrong attestation does not move state to `PendingCet`
    deliver_event!(maker, taker, dummy_wrong_attestation());
    assert_next_state!(CfdState::OpenCommitted, maker, taker, order_id);

    deliver_event!(maker, taker, oracle_data.attestation());
    deliver_event!(maker, taker, Event::CetTimelockExpired(order_id));

    sleep(Duration::from_secs(5)).await;
    assert_next_state!(CfdState::PendingCet, maker, taker, order_id);

    deliver_event!(maker, taker, Event::CetFinality(order_id));
    
    sleep(Duration::from_secs(5)).await; // need to wait a bit until both transition
    assert_next_state!(CfdState::Closed, maker, taker, order_id);
```

The idea is:
- Delivering the two events is related.
- Sleeping before asserting is related.
- Delivering the wrong event is related to the assertion on `OpenCommitted`.

Reply:
I've changed the whitespaces that combines the delivery/sleep/assertion together in a group (the structure changed after your second comment so couldn't just apply your suggestion).